adobe/spectrum-web-components
# feat(menu): convert to core tokens

## Description


Convert to core tokens. Things that still need to be done/looked at:

* [x] checkmark margin via -spectrum-text-to-visual-100 is 8 when it should be 10 to keep the selected item from narrowing
* [x] Menu divider- should it touch the sides in this new version? 
* [x] Menu margins for popover is pretty narrow— but putting a popover on the styling for the css makes it look nicer. can we get clarification on whether or not the sp-menu element should have margins?
* [x] Focus/hover state isn’t getting applied to the menu item background. I think this is due to the “custom vars” 
* [x] add "Sizes" stories to the Menu package
* [x] pass `this.size` down from `Picker.ts` into the `<sp-menu>` child in its shadow DOM
* [x] add a set of "Sizes Open" stories to Picker so that the above change is included in VRTs
* [x] add a set of "Sizes Open" stories to Action Menu and Split Button
* [x] update the VRT cache with the updated and new stories
* [x] get a stable release from the CSS team


## Related issue(s)



- fixes https://github.com/adobe/spectrum-web-components/issues/2254 
- fixes #2984

## Motivation and context


upgrading to core tkoens :) 

## How has this been tested?




    1. Go [here](https://d95f1d9aab4dd919907c3cee1e51bd9b--spectrum-web-components.netlify.app/review/#ActionMenuStories/selects.png)
    2. Compare the changes between the old and new versions. Note the checkmark is now on the left, and that the focus-visible is... well, visible. 

    1. Go [here](https://9905080f40f3bb038d2df931c70e6c4c--spectrum-web-components.netlify.app/review/#ActionMenuStories/selects.png) and see that the WHCM doesn't have that strange black line when a menu item is focused 
    2. Compare the changes between the old and new versions
   
    1. Go [here](https://d95f1d9aab4dd919907c3cee1e51bd9b--spectrum-web-components.netlify.app/review/#MenuStories/MenuGroupSelectsMultiple.png) 
    2. See that the menu divider is visible 
    
     1. Go [here](https://d95f1d9aab4dd919907c3cee1e51bd9b--spectrum-web-components.netlify.app/review/#SubmenuStories/submenu.png) 
     2. Confirm that the space after the chevron is equal to the previous version
## Screenshots (if appropriate)

## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
